### PR TITLE
Added key checking v2

### PR DESCRIPTION
**Pull request by timschumi**

### CODE REVIEW

File: core/main.py [hunk 1]
```diff
@@ -1196,6 +1203,11 @@ def Start_PumpAP(self):
                 'for fix this, You need change on tab (settings -> Class Ranges)'
                 'now you have choose the Class range different of your network.')
 
+        # Check the key
+        if 1 <= self.WPAtype_spinbox.value() <= 2:
```
Comment (mh4x0f) [inline core/main.py:1207]: you need to check if self.GroupApPassphrase is checked in this line.

Comment (timschumi) [reply core/main.py:1207]: Oh, correct. Thanks!

PS: Also, there is no need to merge master into the patching branch.
If you press on the Arrow on the "Merge PR" Button, there should be an option "rebase"

**Comment by timschumi** [comment]
I now have to delete the patch-10 branch on purpose, because my git doesn't seem to like your merge commits

**Comment by mh4x0f** [comment]
update the your fork and send PR No conflict

**Comment by timschumi** [comment]
Sorry, but it did not let me push something to the repository, even after I fetched the changes

**Comment by timschumi** [comment]
The new PR is at #172

**Pull request closed without merging**